src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs

```diff
@@ -19,6 +19,7 @@
 using System.Threading.Tasks;
 using Microsoft.Build.Shared.FileSystem;
 using Microsoft.Build.Utilities;
+using Microsoft.Build.Eventing;
```

Comment:
```suggestion
using Microsoft.Build.Eventing;
using Microsoft.Build.Shared.FileSystem;
using Microsoft.Build.Utilities;
```

Reply:
What about this? Merging the 3 MSBuild using statements with previous block of MSBuild using statements and removing unused Threading statement.
 ```
-using Microsoft.Build.Shared;
+using Microsoft.Build.Eventing;
 using Microsoft.Build.Exceptions;
 using Microsoft.Build.Internal;
-
-using BackendNativeMethods = Microsoft.Build.BackEnd.NativeMethods;
-using System.Threading.Tasks;
+using Microsoft.Build.Shared;
 using Microsoft.Build.Shared.FileSystem;
 using Microsoft.Build.Utilities;
-using Microsoft.Build.Eventing;
+
+using BackendNativeMethods = Microsoft.Build.BackEnd.NativeMethods;
```
(don't know how to properly format it, this is output from git diff)

Reply:
Yes, that looks great.

For formatting, there are two options:

This button:
![image](https://user-images.githubusercontent.com/3347530/87806196-1b3f6380-c81c-11ea-917b-792484c325bc.png)

or starting a code block with the language `diff`

```markdown
```diff
+whatever
-something
```

Reply:
I tried that button, but the result felt confusing. I didn't know about the `diff`, thanks.